### PR TITLE
[2.9] updating dry-run custodian scripts for automation

### PR DESCRIPTION
**Pull request by slickwarren**
## Issue: 

 https://github.com/rancher/qa-tasks/issues/1377

### CODE REVIEW

File: tests/scripts/custodian/Dockerfile [hunk 1]
```diff
@@ -5,39 +5,47 @@ ARG AWS_ACCESS_KEY_ID
...
+
 ADD ./* ./
 SHELL ["/bin/bash", "-c"] 
 
 RUN apt-get update && apt-get install -y python3 python3-pip python3-venv apt-transport-https ca-certificates gnupg curl
 RUN echo "deb [signed-by=/usr/share/keyrings/cloud.google.gpg] http://packages.cloud.google.com/apt cloud-sdk main" | tee -a /etc/apt/sources.list.d/google-cloud-sdk.list && curl https://packages.cloud.google.com/apt/doc/apt-key.gpg | apt-key --keyring /usr/share/keyrings/cloud.google.gpg  add - && apt-get update -y && apt-get install google-cloud-cli -y
 RUN curl -sL https://aka.ms/InstallAzureCLIDeb | bash
-RUN python3.11 -m venv custodian-venv
+
+RUN python3 -m venv custodian-venv
```
Comment (markusewalker) [inline tests/scripts/custodian/Dockerfile:39]: I doubt it, but leaving comment to be sure. We don't think there is going to be an issue with a minor version of `python3` that conflicts, correct?

**Review by markusewalker — COMMENTED** [review]
Overall, looks good, just left one comment.

File: tests/scripts/custodian/Dockerfile (hunk 1 above)
Comment (slickwarren) [reply tests/scripts/custodian/Dockerfile:39]: correct. In fact, adding the minor version was causing an issue 😓

**Review by markusewalker — APPROVED** [review]

**Review by Josh-Diamond — APPROVED** [review]

**Review by susesgartner — COMMENTED** [review]
LGTM

**Pull request merged**